firefox-ios/Client/Frontend/TrackingProtection/TrackigProtectionRedux/TrackingProtectionAction.swift

```diff
@@ -17,6 +17,8 @@ enum TrackingProtectionActionType: ActionType {
     case tappedShowTrackingProtectionDetails
     case tappedShowBlockedTrackers
     case tappedShowClearCookiesAlert
+    case resetReduxStore
```

Comment:
can we rename this as a user / api action as opposed to the consequence of what occurs?